packages/ember-htmlbars/tests/helpers/each_test.js

```diff
@@ -607,6 +612,127 @@ test("it supports {{itemViewClass=}} with in format", function() {
 
 });
 
+test("it supports {{emptyView=}}", function() {
```

Comment:
@jherdman Assigning the container to the view makes a lot more sense then assigning it to the controller (The only place this make sense is on the one test that looks up an item controller).

I made the changes and everything was passing.